src/components/organisms/footer/footer.html

```diff
@@ -0,0 +1,52 @@
+<div class="footer">
+  <div class="footer-title">
+    <img src="../../../img/footer.png" alt="" />
+  </div>
+  <div class="footer-info">
+    〒532-0011 大阪市淀川区西中島6丁目8-23テスト<br />
+    TEL.06-1234-5678 FAX.06-1234-5678 営業時間 / 9:00 - 22:00 定休日 / 不定休
+  </div>
+  <div class="footer-icon">
+    <div class="icon">
+      <i class="fa-brands fa-facebook-f"></i>
+    </div>
+    <div class="icon">
+      <i class="fa-brands fa-twitter"></i>
+    </div>
+    <div class="icon">
+      <i class="fa-brands fa-instagram"></i>
+    </div>
+    <div class="icon">
+      <i class="fa-solid fa-rss"></i>
+    </div>
+  </div>
+  <div class="footer-menu">
+    <ul>
+      <li>
+        <a href="">HOME</a>
+      </li>
+      <li>
+        <a href="">SERVICE</a>
+      </li>
+      <li>
+        <a href="">INTERVIEW</a>
+      </li>
+      <li>
+        <a href="">ACCESS</a>
+      </li>
+      <li>
+        <a href="">CONCEPT</a>
+      </li>
+      <li>
+        <a href="">FLOOR</a>
+      </li>
+      <li>
+        <a href="">BLOG</a>
+      </li>
+      <li>
+        <a href="">DOWNLOAD</a>
+      </li>
+    </ul>
+  </div>
+  <div class="footer-bottom">Copyright © WordPressテーマ「Switch(tcd063)」 All Rights Reserved.</div>
```

Comment:
```suggestion
  <small class="footer-bottom">Copyright © WordPressテーマ「Switch(tcd063)」 All Rights Reserved.</small>
```
著作権表記などのためのタグがあったりもします。わかってる感が出るのでおすすめです。
https://developer.mozilla.org/ja/docs/Web/HTML/Element/small